en/04_Changelogs/5.0.0.md

```diff
@@ -26,6 +28,53 @@ title: 5.0.0 (unreleased)
   - [Other features](#other-features)
 - [Bugfixes](#bugfixes)
 
+## Change to supported modules {#supported-modules}
+
+Some Silverstripe CMS modules are commercially supported. Silverstripe commits to looking after those modules for the duration of the Silverstripe CMS 5 lifecycle.
+
+Review the list of [Silverstripe CMS 5 supported modules](/Project_Governance/Supported_Modules).
+
+### Module not supported in Silverstripe CMS 5
+Some modules that were supported in Silverstripe CMS 4 will not be supported in Silverstripe CMS 5. Just because a module is not supported officially, doesn't mean that you shouldn't be using it. However, it does mean that it's more likely to suffer from regressions. You should take this in consideration when choosing to install an unsupported module in your project.
+
+Some of those modules will provide CMS-5-compatible versions. Others will be dropped altogether.
+
+Email community@silverstripe.org ff you are keen to maintain some of the module that will no longer be officially supported.
+
+Module | Status | Notes
+-- | -- | --
+[bringyourownideas/silverstripe-composer-security-checker](https://packagist.org/packages/bringyourownideas/silverstripe-composer-security-checker) | Dropped | Underlying service does not work anymore.
+[composer/installers](https://github.com/composer/installers) | Dropped | Composer is more established than Silverstripe CMS.
+[cwp-themes/default](https://github.com/silverstripe/cwp-theme-default) | Dropped | 
+[cwp/cwp](https://packagist.org/packages/cwp/cwp) | CMS5 compatible without support | 
+[cwp/cwp-core](https://packagist.org/packages/cwp/cwp-core) | CMS5 comptabile without support | 
+[cwp/cwp-pdfexport](https://packagist.org/packages/cwp/cwp-pdfexport) | Dropped | 
+[cwp/cwp-search](https://packagist.org/packages/cwp/cwp-search) | CMS5 comptabile without support | 
+[hafriedlander/phockito](https://github.com/hafriedlander/phockito) | Dropped | 
+[hafriedlander/silverstripe-phockito](https://packagist.org/packages/hafriedlander/silverstripe-phockito) | Dropped | 
+[lekoala/silverstripe-debugbar](https://packagist.org/packages/lekoala/silverstripe-debugbar) | CMS5 comptabile without support | Debug bar is a development tool that should not be installed in production.
+[silverstripe/activedirectory](https://packagist.org/packages/silverstripe/activedirectory) | Dropped | 
+[silverstripe/akismet](https://packagist.org/packages/silverstripe/akismet) | Dropped | The parent library is outdated and they are better alternatives like [UndefinedOffset/silverstripe-nocaptcha](https://packagist.org/packages/undefinedoffset/silverstripe-nocaptcha)
+[silverstripe/behat-extension](https://github.com/silverstripe/silverstripe-behat-extension) | CMS5 comptabile without official support | 
+[silverstripe/ckan-registry](https://packagist.org/packages/silverstripe/ckan-registry) | CMS5 comptabile without support | 
```

Comment:
Is there a difference between 'without official support' and 'without support'?

Reply:
No there isn't that's typo